src/bindings/interface/importer.i

```diff
@@ -0,0 +1,58 @@
+%module(package="libcellml") importer
+
+#define LIBCELLML_EXPORT
+
+%import "createconstructor.i"
+%import "logger.i"
+%import "types.i"
+
+%feature("docstring") libcellml::Importer
+"Imports CellML objects.";
```

Comment:
CellML components and units, not just any objects.

Reply:
0ecc37d9